package.json

```diff
@@ -10,7 +10,8 @@
     "prop-types": "^15.8.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-scripts": "5.0.1",
+    "react-router-dom": "^6.14.2",
+    "react-scripts": "^2.1.3",
     "uuid": "^9.0.0",
```

Comment:
- [ ] You have done great so far :+1: :+1: But there are some changes you still need to make before moving to the next milestone. :+1: 
- [ ] Looking at the video below, you can see that when I ran the start command, it get an error that prevents the react app from starting, I'll suggest you fix this as your app needs to be able to run properly for users to use it. :+1: :100: 

- - Video reference:

https://github.com/sulaimonaa/math-magicians/assets/107632780/30127635-0ae8-451e-a9a5-c1e81827f81f